### PR TITLE
This code makes the tests first run against fake mode in CI.

### PR DESCRIPTION
**Pull request by TimSimpsonR**

### CODE REVIEW

**Comment by ed-** [comment]
+1

**Comment by hub-cap** [comment]
Sweeeet.

File: scripts/redstack [hunk 1]
```diff
@@ -658,26 +658,41 @@ EOF
...
+
+function init_fake_mode() {
+    CONF_FILE=$PATH_REDDWARF/etc/reddwarf/reddwarf.conf.test
+    cd $PATH_REDDWARF
+    set -e
+    rm -f reddwarf_test.sqlite
+    set +e
+    bin/reddwarf-manage --config-file=$CONF_FILE db_sync \
+        repo_path=reddwarf_test.sqlite
+    sqlite3 reddwarf_test.sqlite \
```
Comment (hub-cap) [inline scripts/redstack:671]: Can we just use the new reddwarf-manage command here?

**Comment by hub-cap** [comment]
change to the reddwarf-manage command and you got yourself a +1

File: scripts/redstack (hunk 1 above)
Comment (hub-cap) [inline scripts/redstack:671]: Also above it, bin/reddwarf-manage, dont we have a reddwarf_manage or some method like that?

Comment (TimSimpsonR) [inline scripts/redstack:671]: Afraid not. For fake mode the config file path is $PATH_REDDWARF/etc/reddwarf/reddwarf.conf.test, while the rd_manager command uses the conf at $USERHOME/reddwarf.conf.

**Comment by rnirmal** [comment]
+1

File: scripts/redstack (hunk 1 above)
Comment (cp16net) [inline scripts/redstack:671]: you can pass in the conf test file though right?

Comment (cp16net) [inline scripts/redstack:671]: ok nm i get it

**Comment by cp16net** [comment]
+1

**Pull request merged**